includes/admin/views/html-notice-update.php

```diff
@@ -18,16 +18,22 @@
 ?>
 <div id="message" class="updated woocommerce-message wc-connect">
 	<p>
-		<strong><?php esc_html_e( 'WooCommerce data update', 'woocommerce' ); ?></strong> &#8211; <?php esc_html_e( 'We need to update your store database to the latest version.', 'woocommerce' ); ?>
+		<strong><?php esc_html_e( 'Database update required', 'woocommerce' ); ?></strong>
```

Comment:
Since this notice can be display on any admin page lets change the title to be more specific, perhaps "WooCommerce database update required". If you run the update routine the heading says "WooCommerce database update" so would be good to match those two.

Reply:
@kloon the next sentance includes "WooCommerce" FYI